### DISCUSSION TITLE
Go-live timing for .NET 8?

### DISCUSSION THREAD

**Discussion by yzorg** [dotnet/runtime#86452] [General] [answered]
When can I start using .NET 8 previews for production?  (I agree to rollback to .NET 7 if there's a big problem.)  I think this is usually called "go-live terms" or "go-live license"?

**Comment by yzorg** [comment]
This is usually with RC1 (in previous years dotnet releases)?

**Reply by teo-tsirpanis** [reply, marked as answer]
Yes. .NET 7 RC1 was released on September 2022, and .NET 8 RC1 will release around September 2023.

**Reply by tannergooding** [reply]
Just noting we can't "guarantee" dates and things are always open to change until the point it actually releases

But, that being said, we are on a decently stable schedule right now and year to year each release is pretty similar to the prior, so you can often expect similar overall timing and dates.
